qiskit/backends/aer/qasm_simulator.py

```diff
@@ -25,6 +25,7 @@
 from qiskit.result._utils import copy_qasm_from_qobj_into_result, result_from_old_style_dict
```

Comment:
this import can go away now.

I think it is fine to basically delete the `copy_qasm_from_qobj_into_result` helper function here. Its main use was to enable the method `Result.get_ran_qasm()`, which has been dead for a while now.

Reply:
Thanks for the clarification Ali!